### ISSUE TITLE
Source Tasks for Few-Shot Experiments

### ISSUE THREAD

**Issue by khalilbalaree** [ZhengxiangShi/DePT#12]
Thanks for the insightful paper! I am currently reproducing the few-shot experiments as reported in the paper and have encountered a point of clarification regarding pretraining prompts on source tasks. Could you please specify which source task the prompts and low-rank pairs were pretrained on for the **CoLA** target task? And what are the **learning steps** and **learning rate** should I use? Thanks for your help.
